src/mongoose.js

```diff
@@ -28,6 +28,10 @@ const ServiceSchema = new Schema({
 	location_id: String,
 	name: {type: String, required: true},
 	name_ES: {type: String},
+	notes: {
```

Comment:
if we have an array named `notes` , then I think that maybe the note would have properties like `text` as opposed to `notes`. you can also add `created_at` since we're keeping track of dates :)

Reply:
Ah. Were you thinking something like an append-only log of notes? If so, then yeah I can change this to an array of objects with text and date properties.

Reply:
that's how I'd interpreted it initially. I was thinking of the manual errors that could occur when multiple people edit the same note over time.  but I thought that was what you were also going after based on your PR. if it was to be a singular item, then the schema would be more like:

```
{
 ...,
    name_ES: {type: String},
    notes: {type: String},
 ...,
}
```

and we'd use a `KeyValue` table that'd look something like the following based on existing practices

```
<KeyValueTable
   rows={[
      {key: 'notes', value: notes},
   ]}
/>
```

`notes` as a property of `notes` just felt wrong to me.

Reply:
I think the array is good. It didn't occur to me to make it a log, but that's much better, and append-only mitigates weird edit conflicts. Something like:

```
{
...,
   notes_log: [
      {
         text: string,
         created_at: Date,
         created_by (maybe?): string
      },
      ...
   ],
...,
}
```

And then yeah, use a KeyValueTable to render it.

Reply:
each note would automatically be given an _id by mongo and the PATCH weirdness is easy enough to circumnavigate. it only updates the changed portion, which would be `notes`. we'd just pass an array of existing + new items each time. see here for the method the PATCH endpoint executes: https://github.com/asylum-connect/catalog-api/blob/0e385e10e3668d362dbe0688aab90893c24dae40/src/routes/organizations.js#L200